b.tsv

```diff
@@ -576,9 +576,9 @@ B48	Sicilian Defense: Taimanov Variation, Bastrikov Variation, English Attack	1.
 B49	Sicilian Defense: Taimanov Variation, Bastrikov Variation	1. e4 c5 2. Nf3 e6 3. d4 cxd4 4. Nxd4 Nc6 5. Nc3 Qc7 6. Be3 a6 7. Be2
 B50	Sicilian Defense	1. e4 c5 2. Nf3 d6 3. d4
 B50	Sicilian Defense	1. e4 c5 2. Nf3 d6 3. d4 cxd4
-B50	Sicilian Defense: Delayed Alapin	1. e4 c5 2. Nf3 d6 3. c3
-B50	Sicilian Defense: Delayed Alapin, Basman-Palatnik Double Gambit	1. e4 c5 2. Nf3 d6 3. c3 Nf6 4. Be2 Nc6 5. d4 cxd4 6. cxd4 Nxe4 7. d5 Qa5+ 8. Nc3 Nxc3 9. bxc3
-B50	Sicilian Defense: Delayed Alapin, Basman-Palatnik Gambit	1. e4 c5 2. Nf3 d6 3. c3 Nf6 4. Be2 Nc6 5. d4 cxd4 6. cxd4 Nxe4
+B50	Sicilian Defense: Delayed Alapin Variation	1. e4 c5 2. Nf3 d6 3. c3
```

Comment:
Looks like the naming inconsistency was hiding an ambiguity (name already exists as `1. e4 c5 2. Nf3 e6 3. c3`). Is one of these uniquely correct? If not, appending  `, with <move>` to both can be used to disambiguate.